src/blocks/alert/Alert.utils.ts

```diff
@@ -0,0 +1,38 @@
+import { AlertVariant } from './Alert.types';
+import { IconProps, InfoFilled, TickCircleFilled, WarningCircleFilled } from '../icons';
+import { ThemeColors } from 'blocks/theme/Theme.types';
+import { FC } from 'react';
+
+export const alertVariants: Record<
+  AlertVariant,
+  { icon: FC<IconProps>; iconColor: ThemeColors; borderColor: ThemeColors; bgColor: ThemeColors; ctaColor: ThemeColors }
+> = {
+  success: {
+    icon: TickCircleFilled,
+    iconColor: 'components-alert-icon-success',
+    borderColor: 'components-alert-stroke-success',
+    bgColor: 'components-alert-background-success',
+    ctaColor: 'components-alert-text-cta-success',
+  },
+  warning: {
+    icon: WarningCircleFilled,
+    iconColor: 'components-alert-icon-warning',
+    borderColor: 'components-alert-stroke-warning',
+    bgColor: 'components-alert-background-warning',
+    ctaColor: 'components-alert-text-cta-warning',
```

Comment:
Check text area component for the color implementation for different variants. This constant isnt needed, it can be done in a much optimised way

Reply:
This is similar to how we have implemented `alertModalIcon` in the AlertModal.